### PR TITLE
added homescreen tests

### PR DESCRIPTION
**Pull request by nzurnic3**

### CODE REVIEW

File: src/main/java/rs/urosdragojevic/emotionsters/controller/HomeScreenController.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import rs.urosdragojevic.emotionsters.model.HomeScreen;
+import rs.urosdragojevic.emotionsters.service.HomeScreenService;
+
+@RestController
+@RequestMapping(value = "/emotionsters")
```
Comment (urosdragojevic) [inline src/main/java/rs/urosdragojevic/emotionsters/controller/HomeScreenController.java:11]: We don't really need this /emotionsters in the path, but you can keep it or remove it

File: src/test/java/rs/urosdragojevic/emotionsters/EmotionstersApplicationTests.java [hunk 2]
```diff
@@ -10,4 +10,9 @@ class EmotionstersApplicationTests {
 	void contextLoads() {
 	}
 
+	@Test
+	void firstMessageSerbian(){
+//		assert ()
```
Comment (urosdragojevic) [inline src/test/java/rs/urosdragojevic/emotionsters/EmotionstersApplicationTests.java:15]: Remove empty test

File: src/test/java/rs/urosdragojevic/emotionsters/controller/HomeScreenControllerTests.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+@SpringBootTest
+class HomeScreenControllerTests {
+
+    @Test
+    void homeScreenMessage() throws IOException, ParseException {
+        HttpUriRequest request = new HttpGet("http://localhost:8080/emotionsters/homeScreenSpeechBubbles");
+        CloseableHttpResponse httpResponse = HttpClientBuilder.create().build().execute(request);
+        String jsonFromResponse = EntityUtils.toString(httpResponse.getEntity());
+
+        assertEquals("{\"jacksTopBubble\":\"Hi, I'm Jack. Welcome to Monster Kingdom!\"," +
```
Comment (urosdragojevic) [inline src/test/java/rs/urosdragojevic/emotionsters/controller/HomeScreenControllerTests.java:25]: Extract the string here into a variable e.g. `String expected`, then use that in the `assertEquals(expected, jsonFromResponse)`

File: src/test/java/rs/urosdragojevic/emotionsters/service/HomeScreenServiceTests.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class HomeScreenServiceTests {
+
+    private final HomeScreenService homeScreenService = new HomeScreenServiceImpl();
+
+    @Test
+    void getTextForTopSpeechBubbleOnHomeScreen() {
+        assertEquals("Hi, I'm Jack. Welcome to Monster Kingdom!", homeScreenService.homeScreenSpeechBubbles().getJacksTopBubble());
```
Comment (urosdragojevic) [inline src/test/java/rs/urosdragojevic/emotionsters/service/HomeScreenServiceTests.java]: You can do the same as above for these. Create constants like jacksTopBubbleExpected and jacksBottomBubbleExpected and use that in assertEquals

File: pom.xml [hunk 5]
```diff
@@ -57,12 +53,23 @@
...
 			<scope>test</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.springframework.security</groupId>
-			<artifactId>spring-security-test</artifactId>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>com.vaadin.external.google</groupId>
```
Comment (urosdragojevic) [inline pom.xml:57]: Do we need all these dependencies here?

**Review by urosdragojevic — CHANGES_REQUESTED** [review]

File: src/main/java/rs/urosdragojevic/emotionsters/controller/HomeScreenController.java (hunk 1 above)
Comment (nzurnic3) [reply src/main/java/rs/urosdragojevic/emotionsters/controller/HomeScreenController.java:11]: Keep it for now

File: src/test/java/rs/urosdragojevic/emotionsters/EmotionstersApplicationTests.java (hunk 2 above)
Comment (nzurnic3) [reply src/test/java/rs/urosdragojevic/emotionsters/EmotionstersApplicationTests.java:15]: Done

File: src/test/java/rs/urosdragojevic/emotionsters/controller/HomeScreenControllerTests.java (hunk 3 above)
Comment (nzurnic3) [reply src/test/java/rs/urosdragojevic/emotionsters/controller/HomeScreenControllerTests.java:25]: Done

File: src/test/java/rs/urosdragojevic/emotionsters/service/HomeScreenServiceTests.java (hunk 4 above)
Comment (nzurnic3) [reply src/test/java/rs/urosdragojevic/emotionsters/service/HomeScreenServiceTests.java]: Done

File: pom.xml (hunk 5 above)
Comment (nzurnic3) [reply pom.xml:57]: I don't think we do, deleted.

**Review by urosdragojevic — APPROVED** [review]

**Pull request merged**